tests/acceptance/test_inventory.py

```diff
@@ -23,6 +23,7 @@
 
 @pytest.mark.usefixtures("setup_board", "bitbake_path")
 class TestInventory:
+    @pytest.mark.cross_platform
```

Comment:
Keep this one, it's testing that our inventory script is returning the correct filesystem type, very relevant for UBI.